src/components/admin/services/timer/TimerControls.jsx

```diff
@@ -0,0 +1,68 @@
+import {
+  FaPlay,
+  FaPause,
+  FaUndo,
+  FaTrashAlt,
+  FaPen,
+  FaCheck,
+  FaTimes,
+} from "react-icons/fa";
+import Tooltip from "./Tooltip";
+
+const TimerControls = ({
+  isEditMode,
```

Comment:
surely we do not need these many props for each one 😭 is it possible we can reduce this list